oshdb-api/src/test/java/org/heigit/ohsome/oshdb/api/tests/TestOhsomeFilter.java

```diff
@@ -57,8 +57,7 @@ public void testFilterString() throws Exception {
 
   @Test
   public void testFilterObject() throws Exception {
-    MapReducer<OSMEntitySnapshot> mr = createMapReducer();
-    Number result = mr
+    Number result = createMapReducer()
```

Comment:
Is this a cleanup that changes or affects anything apart from syntax cleanup?

Reply:
just a minor cleanup, when the test only needs one mapreducer, we don't need the local variable. :shrug: